dspace-uclouvain/src/main/java/org/dspace/uclouvain/authority/UCLouvainAbstractAuthority.java

```diff
@@ -0,0 +1,76 @@
+package org.dspace.uclouvain.authority;
+
+import java.util.Map;
+
+import org.dspace.content.authority.Choice;
+import org.dspace.content.authority.ChoiceAuthority;
+import org.dspace.content.authority.Choices;
+import org.dspace.uclouvain.external.dilbert.model.DialPerson;
+
+public abstract class UCLouvainAbstractAuthority implements ChoiceAuthority {
```

Comment:
In Java best practice, the "Abstract" word is the first of the class name --> "AbstractUCLouvainAuthority"